src/ir_LG.cpp

```diff
@@ -329,10 +352,22 @@ void IRLgAc::setRaw(const uint32_t new_code, const decode_type_t protocol) {
       break;
   }
   // Look for model specific settings/features to improve model detection.
-  if (_isAKB74955603()) setModel(lg_ac_remote_model_t::AKB74955603);
+  if (_isAKB74955603()) {
+    setModel(lg_ac_remote_model_t::AKB74955603);
+    if (isSwingV()) _swingv = new_code;
+  }
+  if (_isAKB73757604()) {
+    setModel(lg_ac_remote_model_t::AKB73757604);
+    if (isVaneSwingV()) {
+      // Extract just the vane nr and position part of the message.
+      const uint32_t vanecode = getVaneCode(_.raw);
+      _vaneswingv[vanecode / kLgAcVaneSwingVSize] = vanecode & 0xF;
```

Comment:
this vanecode `& 0xF` exists twice, could a macro be used as a way of documentation?